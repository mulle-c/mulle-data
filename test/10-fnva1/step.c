#include <mulle-data/mulle-data.h>
#include <stdio.h>


int   main( void)
{
   uint32_t   hash;
   char       *s;

   hash = MULLE_FNV1A_32_INIT;
   s    = "VfL Bochum 1848";
   while( *s)
   {
      printf( "%08x + %c", hash, *s);
      hash = _mulle_fnv1a_step_32( hash, *s);
      printf( "-> %08x\n", hash);
      ++s;
   }
   return( 0);
}