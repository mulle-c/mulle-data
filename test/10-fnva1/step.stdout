811c9dc5 + V-> d30c0e69
d30c0e69 + f-> 4afa219d
4afa219d + L-> d8c33c03
d8c33c03 +  -> 5e57ab19
5e57ab19 + B-> df02c041
df02c041 + o-> 3f54886a
3f54886a + c-> bb12262b
bb12262b + h-> c0923b79
c0923b79 + u-> 3232f3e4
3232f3e4 + m-> 8f3560ab
8f3560ab +  -> fc06fad1
fc06fad1 + 1-> 9efceea0
9efceea0 + 8-> e02b9948
e02b9948 + 4-> 60a29e34
60a29e34 + 8-> 2bfecce4
